### PR TITLE
Fix segmentation (only works for landscape view, not for portrait)

### PR DESCRIPTION
**Pull request by Mati-AAU**

### CODE REVIEW

File: app/src/main/res/raw/segment_postprocess.glsl [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#extension GL_OES_EGL_image_external : require
```
Comment (mads256h) [inline app/src/main/res/raw/segment_postprocess.glsl:1]: Rename this file to segment_postprocess_shader.glsl

**Review by mads256h — DISMISSED** [review]

File: app/src/main/java/dk/scuffed/whiteboardapp/segmentation/PPSegmentation.kt [hunk 2]
```diff
@@ -20,8 +20,11 @@ class PPSegmentation (context : Context, private val model : Model) {
     private val segmentationResult : SegmentationResult = SegmentationResult()
 
     fun segment(input : Bitmap) : Bitmap{
-        // predict on the downscaled input
-        segmentationModel.predict(input.scale(model.width, model.height), segmentationResult)
+        if (input.width !=  model.width || input.height !=  model.height){
+            throw java.lang.Exception("Resolution of input bitmap to segmentor ("+input.width + "," + input.height +") doesnt match required resolution of("+model.width + "," + model.height +")");
```
[the remark this answers is not in the document]
Comment (Mati-AAU) [reply app/src/main/java/dk/scuffed/whiteboardapp/segmentation/PPSegmentation.kt]: done

File: app/src/main/res/raw/segment_postprocess.glsl (hunk 1 above)
Comment (Mati-AAU) [reply app/src/main/res/raw/segment_postprocess.glsl]: done

**Review by Casper-NS — APPROVED** [review]
lgtm

**Review by mads256h — APPROVED** [review]

**Pull request merged**